topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,534 @@
+---
+layout: tutorial_hands_on
+title: "Downstream Single-cell RNA Plant analysis with ScanPy"
+subtopic: single-cell
+priority:
+
+zenodo_link: 'https://zenodo.org/record/4597857'
+tags:
+  - single-cell
+questions:
+  - Can we reclaim cell markers using a different analysis method?
+  - Are highly variable genes paramount to the analysis?
+objectives:
+  - Perform filtering, dimension reduction, and clustering
+  - Generate a DotPlot emulating the original paper using a different analysis tool
+  - Determine robust clusters across scRNA-seq pipelines
+requirements:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-scanpy-pbmc3k
+
+time_estimation: 2H
+key_points:
+  - Filtering parameters are dependent on the dataset, and should be explored using scatter or violin plots
+  - A DotPlot is a fantastic way to validate clusters across different analyses
+contributors:
+  - mtekman
+  - beatrizserrano
+
+
+gitter: Galaxy-Training-Network/galaxy-single-cell
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+> ### {% icon comment %} Comment
+>
+> Please familiarise yourself with the ["Clustering 3K PBMCs with ScanPy"]({% link topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md %}) tutorial first, as much of the process is the same, and the accompanying slide deck better explains some of the methods and concepts better.
+>
+{: .comment}
```

Comment:
I think this might fit better next to `# Data`, either before or after. No sense warning them before they've read the summary and know what it's about.